# curl releases after 7.72.0 changed to a shorter help list.
# Upstream fixed Curl.pm in 2.9.0 but not AnotherCurl.pm.
--- a/lib/Smokeping/probes/AnotherCurl.pm
+++ b/lib/Smokeping/probes/AnotherCurl.pm
@@ -216,7 +216,7 @@ sub test_usage {
 
 	my $arghashref = $self->features;
 	my %arghash = %$arghashref;
-        my $curl_man = `$bin --help`;
+        my $curl_man = `$bin --help all`;
         
 	for my $feature (keys %arghash) {
 		next if $curl_man =~ /\Q$arghash{$feature}/;
